src/ingest-pipeline/misc/tools/airflow_wrapper.sh

```diff
@@ -84,13 +70,15 @@ for varname in "${envvars[@]}" ; do
 done
 
 if [ "${HM_AF_METHOD}" == 'conda' ] ; then
-    export PATH=/hive/users/hive/anaconda3/bin:%PATH
-    export LANG=C.UTF-8
-    export LC_ALL=C.UTF-8
+    which conda || export PATH=/hive/users/hive/anaconda3/bin:$PATH
+    #export LC_ALL=C.UTF-8
```

Comment:
@sunset666 I notice you added the LC_ALL and LANG exports as a bugfix, but my experiments on DEV showed they actually caused errors during start-up.  I'm concerned that the need for these lines may be host-dependent.